runtime/tutor

```diff
@@ -986,7 +986,9 @@ lines.
  --> if a wouldchuck could chuck would?
 
  Note: Additionally, Alt-( and Alt-) cycle the *contents* of the
-       selections as well.
+       selections as well. To see this in action, first select
+       different contents, like the words "would" and "much"
+       of the first line.
```

Comment:
Each section of the tutor has a fixed height so these additions won't fit. It might be worth having a separate section for `A-(`/`A-)`

Reply:
Done with commit [ee7092d](https://github.com/helix-editor/helix/pull/5161/commits/ee7092d0c4223bf52db7a38af1a1641b59f03f22)